documentation/src/main/asciidoc/topics/dependencies_maven/jcache.xml

```diff
@@ -3,3 +3,7 @@
   <artifactId>infinispan-jcache</artifactId>
   <version>${version.infinispan}</version>
 </dependency>
+<dependency>
```

Comment:
I'm not sure about this... 
`cache-api` must be in the classpath if JCache is being used, independently of the provider used.

Reply:
removed this from the pom example and added details about putting `cache-api` on the classpath.